spaceship/templates/base_with_quill.html

```diff
@@ -108,7 +123,7 @@ <h5 class="modal-title" v-html="modal_title"></h5>
 
       this.savedStates['invite'] = {
         modal_title: "Invite Crew",
-        subject: "Join my Spaceship Earth crew!",
+        subject: "{{ invitation_subject }}",
```

Comment:
escaped for js?

Reply:
mmm the default is html-safe, so html entities will show up ugly. is html-safe a superset of js-safe?

Reply:
i was more thinking about ".  random people on the internet say to do (no quotes) {{ invitation_subject|tojson }} in jinja2